.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
How ironic, he could save others but not himself

Reply:
> Perhaps 3 spaces indentation instead of 4? Also I like trailing commas for code review. Please consider :)

Agreed, added trailing commas